app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
```

Comment:
why do we need cleartext traffic? This might be a security issue

Reply:
this is only trafic toward localhost, therefore the security risk is extremely low, since you would have a network sniffer on your defice to see this traffic ... for this you require root ... if somebody got this far everything is lost anyways ... right?

Reply:
Also this was not enforced on all android version, had to add it for newer versions. 

Why we want this: This allows us to directly communicate via http to other apps on this device (e.g. a WalletConnect server embedded into an app). This is safes data compared to routing via an external service. But normally these apps don't ship their own ssl certificate (would also not make much sense, since you would kind of expose the secret part of it). Therefore we allow http for localhost.